### ISSUE TITLE
No module named 'win32gui'

### ISSUE THREAD

**Issue by Enchante503** [toyxyz/ComfyUI_toyxyz_test_nodes#48]
I'm using WSL2 Ubuntu 22.04.4 LTS

I'm having trouble with being asked for modules for Windows
How can I get around this?

Why not write code to detect the platform and get around it?

```
/home/user/ComfyUI/custom_nodes/ComfyUI_toyxyz_test_nodes/toyxyz_test_nodes.py", line 14, in <module>
    import win32gui
ModuleNotFoundError: No module named 'win32gui'

Warn!: Cannot import /home/user/ComfyUI/custom_nodes/ComfyUI_toyxyz_test_nodes module for custom nodes: No module named 'win32gui'
```

**Comment by toyxyz** [comment]
I don't know enough about WSL and Ubuntu to give you an exact solution, but you can try `pip install pywin32`

**Comment by toyxyz** [comment]
> Hi, Since WSL2 is Linux, the win32 (pywin32) module cannot be used.
> 
> Is ComfyUI_toyxyz_test_nodes used only for Windows environments?

Yes. Tested on Windows environments only. I don't know much about Linux, so it's hard to help.
